app/controllers/students/students_learning_resources_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Students
+  class StudentsLearningResourcesController < ApplicationController
+    before_action :authenticate_user!, :set_user
+
+    def index
+      @student_learning_resources = @user.students_learning_resources
```

Comment:
Something doesn't feel right with this instance method `students_learning_resources`
This returns a list of learning resources for the specific student, right?

Well two thing:
1 - Should be `@student.students_learning_resources`, which sounds redundant
2 - There's **one** instance, method name should be **singular**, otherwise I'd expect a collection of learning resources for all students, not the current one